includes/class-wc-stripe-payment-tokens.php

```diff
@@ -392,7 +392,11 @@ public function get_account_saved_payment_methods_list_item( $item, $payment_tok
 				$item['method']['brand'] = esc_html__( 'Cash App Pay', 'woocommerce-gateway-stripe' );
 				break;
 			case WC_Stripe_Payment_Methods::LINK:
-				$item['method']['brand'] = esc_html__( 'Stripe Link', 'woocommerce-gateway-stripe' );
+				$item['method']['brand'] = sprintf(
```

Comment:
This string should be escaped after the `sprintf()`.

Even though WooCommerce escapes this in their template `woocommerce/templates/myaccount/payment-methods.php` we can't guarantee this for all stores/themes that override this template 😞

Reply:
Good call, escaping added in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3641/commits/ec8707dfbb7d0884af7ad95a4ebee3a9552a74da